ci/lint/check-documentation-style.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+wget https://github.com/errata-ai/vale/releases/download/v2.28.0/vale_2.28.0_Linux_64-bit.tar.gz
+tar -xvzf vale_2.28.0_Linux_64-bit.tar.gz -C .
```

Comment:
only extract the `vale` binary?

also, you should probably `mktemp -d` a directory and extract there, rather than doing it here.

and remove that directory after the script execution.